api/types/types.proto

```diff
@@ -1543,3 +1543,29 @@ message DeleteWebTokenRequest {
     // Token specifies the token to delete.
     string Token = 2 [ (gogoproto.jsontag) = "token" ];
 }
+
+// GetResourceRequest is a request to query a resource.
+message GetResourceRequest {
+    // Name is the name of the resource to get.
+    string Name = 1 [ (gogoproto.jsontag) = "name" ];
+}
```

Comment:
This and `DeleteResourceRequest` are basically the same.
Can they be combined into `message Resource` or `NamedResource`?